frontend/src/app/api/addToSheet/route.ts

```diff
@@ -0,0 +1,35 @@
+import { NextResponse } from 'next/server';
+import { google } from 'googleapis';
+
+export async function POST(request: Request) {
+  try {
+    const body = await request.json();
+    const { name, email, role } = body;
+
+    const auth = new google.auth.GoogleAuth({
+      credentials: {
+        client_email: process.env.GOOGLE_CLIENT_EMAIL,
+        private_key: process.env.GOOGLE_PRIVATE_KEY?.replace(/\\n/g, '\n'),
+      },
```

Comment:
please dont forget to add an example env file for this

Reply:
I added examples to .env.example and also a separate guide to `docs` dir